NuRadioReco/detector/test/tests.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 set -e
 
-python3 NuRadioReco/detector/test/test_rnog_detector.py
\ No newline at end of file
+python3 NuRadioReco/detector/test/test_rnog_detector.py
+python3 NuRadioReco/detector/test/test_response.py
```

Comment:
I think this won't work reliably - if the first python script raises an error, the second one will probably still run, and the github test will only consider the output of the second test. Feel free to put this into a new test, I think.

Reply:
I see thanks! I fixed it in the workflow yaml. Would that work?